client/orchestrator/client_orchestrator.go

```diff
@@ -58,6 +59,20 @@ type ProvisionDeviceReq struct {
 	Device model.Device `json:"device"`
 }
 
+// UpdateDeviceStatusReq contains request data of request to start update
+// device status  workflow
+type UpdateDeviceStatusReq struct {
+	// Request ID
+	RequestId string `json:"request_id"`
+	// Device IDs
+	//Ids []string `json:"device_ids"`
+	Ids string `json:"device_ids"`
```

Comment:
it's either a single Id, or a list of Ids.

what's the backstory? in what scenario is it useful to update the status of multiple devices?

Reply:
the back story is: for now we only update a single device. but we have to change the UI to update multiple devices at once. so the workflow supports that, but you have to pass to the workflow a valid list in json, since there is no place to Marshal the arguments in the workflows, this has to be Marshalled [here](https://github.com/mendersoftware/deviceauth/pull/324/commits/d55f727cfd92f5ed7328cab221616abc57c9d03c#diff-bafc5a98b88ba2baa09868cc001b23e0R473)
So the workflow is as follows:
1. device changes state
2. deviceauth creates one element array with the device id, marshals it into json string
3. deviceauth calls update_device_status workflow which passes the string from 2. containing json array of ids as a workflow [parameter](https://github.com/mendersoftware/inventory/pull/162/commits/1c00753a07a513366699ed866638060a788267aa#diff-d38bb1bb13c34ab0a761f45bb70cd491R623) into POST call to internal inventory [endpoint](https://github.com/mendersoftware/inventory/pull/162/commits/1c00753a07a513366699ed866638060a788267aa#diff-d38bb1bb13c34ab0a761f45bb70cd491R623).
At one point the device auth will have to be called from UI with multiple devices to update (that is what the endpoint I had added here was supposed to be doing.)